app/views/course/groups/edit.html.slim

```diff
@@ -0,0 +1,8 @@
+h2= t('.header')
+
+= simple_form_for [current_course, @group], resource: :course_group do |f|
```

Comment:
As I said in the PR description, I'll implement group user management later. So that time edit and new group will be different, that's why I didn't put this in a partial.